### DISCUSSION TITLE
How to assembly and connect the speedometer?

### DISCUSSION THREAD

**Discussion by Gabin37** [CircuitSetup/Time-Circuits-Display#49] [General]
Hi everyone, how do you connect the speedometer to the TCD please? The ending of the cable is a XH 4 pin and I can see that the Speedometer borner on the TCD is a green block.. How does it work please? Is there any changes to apply on the web portal? Thanks a lot for your help, Regards

**Comment by realA10001986** [comment]
https://github.com/realA10001986/Time-Circuits-Display/tree/main#speedometer

**Comment by Gabin37** [comment]
Hi, here is a video, could you please tell me where I am wrong? 

https://github.com/CircuitSetup/Time-Circuits-Display/assets/154004044/7a07fbee-870a-45a1-802a-4f17cefec781

**Reply by Gabin37** [reply]
The thing is that I do have the green adapter that is on the speedo green connector on the control panel. So, I did the wiring both sides of the cable. But But when I plug the both sides to the speedometer and the control panel, the TCD stops working and seems to be blocked as you can see on the video attached.

And I chose “circuit [setups.us](http://setups.us/)” as speedo display type though

**Reply by CircuitSetup** [reply]
> The thing is that I do have the green adapter that is on the speedo green connector on the control panel. So, I did the wiring both sides of the cable. But But when I plug the both sides to the speedometer and the control panel, the TCD stops working and seems to be blocked as you can see on the video attached.
> 
> And I chose “circuit [setups.us](http://setups.us/)” as speedo display type though

You won't need the adapter. Earlier versions of the control board did not have the green connector on the PCB. 

Plug in the Speedo before turning everything on. It reads if it is there or not on power up.

Also, it's hard to tell but I think you have the orange/white wires switched. Pin 2 (from the left) should be orange with white stripe. Pin 4 should be white with orange stripe.

**Comment by Gabin37** [comment]
Yes!!! I did not see the différence between the orange/white wires!Now it works perfectly! :-)))Thanks a lot for your help ! Best regards GabinLe 14 juin 2024 à 14:58, John ***@***.***> a écrit :﻿

The thing is that I do have the green adapter that is on the speedo green connector on the control panel. So, I did the wiring both sides of the cable. But But when I plug the both sides to the speedometer and the control panel, the TCD stops working and seems to be blocked as you can see on the video attached.
And I chose “circuit setups.us” as speedo display type though

You won't need the adapter. Earlier versions of the control board did not have the green connector on the PCB.
Plug in the Speedo before turning everything on. It reads if it is there or not on power up.
Also, it's hard to tell but I think you have the orange/white wires switched. Pin 2 (from the left) should be orange with white stripe. Pin 4 should be white with orange stripe.

**Comment by Gabin37** [comment]
Hi John,

I’ve just realized I lost the the plastic bag containing the «  set, to, 88 » stickers….
Could you please, send it again for me please? 

Of course I pay the shipment, as I live in France.

Let me know.

Best regards,

Gabin

**Comment by Gabin37** [comment]
anyone? :)
